bdd/spec/011_array_spec.sh

```diff
@@ -232,7 +232,7 @@ Hello, World!"
 
         on start {
           const count = [1, 2, 3, 4, 5]; // Ah, ah, ahh!
-          const byTwos = count.map(fn (n: int64): int64 = n * 2);
+          const byTwos = count.map(fn (n: int64): int64 = n * 2 || 0);
```

Comment:
This keeps the types the same, but I think if I just popped off the return type from this map function and removed the typed wrapper function around `toString` below it will also just work since you can print `Result<Stringifiable>` types.

Debating which to do here.